src/compiler/checker.ts

```diff
@@ -622,7 +622,7 @@ module ts {
             if (!links.referenced) {
                 links.referenced = true;
                 var node = getDeclarationOfAliasSymbol(symbol);
-                if (node.kind === SyntaxKind.ExportAssignment) {
+                if (node.kind === SyntaxKind.ExportAssignment && (<ExportAssignment>node).expression) {
                     // export default <symbol>
                     checkExpressionCached((<ExportAssignment>node).expression);

```

Comment:
Or checkExpressionCached could be resilient to an undefined argument. But I don't have a strong preference either way.